src/Interfaces/AbstractPolyhedron_functions.jl

```diff
@@ -879,3 +879,42 @@ function an_element(P::AbstractPolyhedron{N}) where {N<:Real}
     e₁ = SingleEntryVector(1, n, one(N))
     return σ(e₁, P)
 end
+
+"""
+    isbounded(P::AbstractPolyhedron{N}; solver=LazySets.default_lp_solver(N)) where {N}
+
+Determine whether a polyhedron is bounded.
+
+### Input
+
+- `P`       -- polyhedron
+- `backend` -- (optional, default: `default_lp_solver(N)`) the backend used
+               to solve the linear program
+
+### Output
+
+`true` iff the polyhedron is bounded
+
+### Algorithm
+
+The algorithm is based on Stiemke's theorem of alternatives.
```

Comment:
Can you give a reference?

Reply:
the algorithm can be seen as a corollary on the theorem of alternatives (see [Farkas lemma](https://en.wikipedia.org/wiki/Farkas%27_lemma), though i don't have a specific reference for it. about the particular fm of the theorem of alternatives, there is 

Mangasarian, Olvi L. Nonlinear programming. Society for Industrial and Applied Mathematics, 1994.